src/helm/benchmark/run_spec_factory.py

```diff
@@ -138,6 +138,13 @@ def alter_run_spec(run_spec: RunSpec) -> RunSpec:
         ):
             run_spec = singleton(IncreaseMaxTokensRunExpander(value=1).expand(run_spec))
 
+        if model.name == "openai/o1-2024-12-17":
```

Comment:
@yifanmai `o1-2024-12-17` specifically is unusable in HELM without this change.

Reply:
In practice a few thousand reasoning tokens are used at least for the VHELM scenarios.